src/app/main-overview/components/ZoneControlPlanesDetails.vue

```diff
@@ -55,7 +55,13 @@ const props = defineProps({
 const tableData = computed(() => props.zoneOverviews.map((zoneOverview) => {
   const { name } = zoneOverview
 
-  const status = getItemStatusFromInsight(zoneOverview.zoneInsight)
+  let status: StatusKeyword | 'disabled'
+
+  if (zoneOverview.zone.enabled === false) {
+    status = 'disabled'
+  } else {
+    status = getItemStatusFromInsight(zoneOverview.zoneInsight)
+  }
```

Comment:
nitpick: if I read it correctly and by default it's disabled then why do we need to set it to disabled one more time? can we just do:

```suggestion
  if (zoneOverview.zone.enabled === true) {
    status = getItemStatusFromInsight(zoneOverview.zoneInsight)
  }
```

Reply:
There is no default value in this context currently. Perhaps you read the type annotation above wrong?

Current:

```ts
  let status: StatusKeyword | 'disabled'

  if (zoneOverview.zone.enabled === false) {
    status = 'disabled'
  } else {
    status = getItemStatusFromInsight(zoneOverview.zoneInsight)
  }
```

We can however do what you suggest:

```ts
  let status: StatusKeyword | 'disabled' = 'disabled'

  if (zoneOverview.zone.enabled === true) {
    status = getItemStatusFromInsight(zoneOverview.zoneInsight)
  }
```

I made those changes in the two relevant places now.